src/hooks/loginHooks/useSgidLogin.ts

```diff
@@ -3,17 +3,21 @@ import { useQuery, UseQueryResult } from "react-query"
 
 import * as LoginService from "services/LoginService"
 
-import { VerifySgidLoginParams } from "types/login"
+import { PublicOfficerData, VerifySgidLoginParams } from "types/login"
 
 export const useSgidLogin = (
   body: VerifySgidLoginParams
-): UseQueryResult<void> => {
-  return useQuery([body], () => LoginService.verifySgidLogin(body), {
-    onSuccess: () => {
-      window.location.replace("/sites")
-    },
-    onError: (err: AxiosError) => {
-      window.location.replace(`/ogp-login?status=${err.response?.status}`)
-    },
-  })
+): UseQueryResult<PublicOfficerData[]> => {
+  return useQuery(
+    [body],
+    () =>
+      LoginService.verifySgidLogin(body).then((data) => {
+        return data.userData
+      }),
+    {
+      onError: (err: AxiosError) => {
+        window.location.replace(`/ogp-login?status=${err.response?.status}`)
```

Comment:
good catch - how did the video work though

Reply:
Good catch! swapped in 0ac89096

it worked in the video because those show the happy paths, while only the sad paths redirect back to the login page